README.md

```diff
@@ -1 +1,21 @@
 # Governance and Guidance
+
+This Repository hosts the collection of documents that define how the organization of Tauri-Apps works and understands itself. Unless these documents are clearly superceded in other repositories within this organization's collection, they are considered binding. 
+
+## Document List
+
+ORGANIZATION
+ - CODE OF CONDUCT
+ - SOCIAL CONTRACT
+ - WORKING GROUPS
+ 
+CODE
+ - LICENSE
+ - CONTRIBUTOR LICENSE AGREEMENT
```

Comment:
if any

Reply:
yes - i was hoping this would be a "prod" to have the discussion. :)